### ISSUE TITLE
gateio and okex both have symbol AE (AE/USDT), but for different things

### ISSUE THREAD

**Issue by npomfret** [ccxt/ccxt#4981]
Looks like OKEx is referring to the ERC20 token and gateio is referring the the mainnet currency.

https://coinmarketcap.com/currencies/aeternity/

Perhaps we should rename the symbol on OKEx to AET?

**Comment by kroitor** [comment]
References:

- https://support.okex.com/hc/en-us/articles/360025353231-Aeternity-AE-Mainnet-Swap-Depositing-and-Withdrawal-Suspension

<img width="1194" alt="Screen Shot 2019-04-12 at 20 08 53" src="https://user-images.githubusercontent.com/1294454/56054207-de560880-5d5e-11e9-99e5-b12d85b5ae1c.png">


Thx for reporting!

**Comment by kroitor** [comment]
@npomfret can you plz provide your sources or links for this:

> Looks like OKEx is referring to the ERC20 token

?

**Comment by npomfret** [comment]
Because it says "token deposit" on their site.  And when I tried moving some AE from gateio to okex I got the error message "Please use AE mainnet address but not ERC20".

Additionally, the price is about 6% different, so it makes sense that they aren't the same thing.

**Comment by kroitor** [comment]
@npomfret thx again! Appreciate your help!

**Comment by npomfret** [comment]
The other thing is that on gateio the wallet has the currency name instead of the symbol and it says _AE Aeternity_ (and doesn't mention token).  I'm 100% sure these assists aren't fungible.  I believe one must be the currency and the other the token.

**Comment by kroitor** [comment]
@npomfret 
> I'm 100% sure these assists aren't fungible. I believe one must be the currency and the other the token.

Pardon, can you plz elaborate on this? Did I miss something? Not sure that I completely got this one.

**Comment by npomfret** [comment]
I believe Aeternity had a token before their mainnet launch.  The token is trading on okex and the main currency is traded on gateio.  I think!

I don't believe that the token and the currency are interchangeable.  When I tried to move my AE from gateio to okex I got ab error message saying _Please use AE mainnet address but not ERC20_ 

So I believe the okex AE markets are actually the token (which we could call AET).
